Add page for extpar and blog post (#255)
C2SM/c2sm.github.io: docs/posts/2025-02-20_extpar.md

---
date:
  created: 2025-02-20
categories:
  - ICON
  - EXTPAR
---

# EXTPAR: New release v5.15 and documentation available

> A new release of the [EXTPAR](../tools/extpar.md) software has been published today. This comes along with several 
> bugfixes, new features, automatic container deployment and a new documentation. 

Reviewer: ```suggestion
A new release of the [EXTPAR](../tools/extpar.md) software has been published today. It includes several bug fixes, new features, automatic container deployment and a new documentation. 
```

<!-- more -->

> The documentation is now publicly available and has been compiled from several README files and the “User and Implementation Guide”, the existing LaTeX document that was previously made available as a PDF with each new version. Now it has all the advantages of a documentation page, such as better readability, search function, section links, etc.
> 

Reviewer: ```suggestion
The documentation is now public. It combines several README files and the "User and Implementation Guide", which was previously a LaTeX PDF included with each release. It now has all the advantages of a documentation page, such as better readability, search function, section links, etc.

```
